src/common-lib/urls/betaStage.ts

```diff
@@ -0,0 +1,2 @@
+export const BETA_STAGE = ["beta", "gamma"] as const;
```

Comment:
Is the beta/gamma distinction still used in addition to teachers/teachers-2023?

Reply:
whoops, left that in unintentionally. No more gamma